index.html

```diff
@@ -212,293 +234,583 @@ <h2>Dependencies</h2>
+        <p>
+        The REQUIRED <dfn><code>max_age</code></dfn> member specifies the
+        lifetime of this <a>NEL policy</a>, as a non-negative integer number of
+        seconds.  Its value MUST be an non-negative integer; any other type will
+        result in a parse error.
+        </p>
 
-          <dt>unknown</dt>
-          <dd>error type is unknown</dd>
-        </dl>
+        <p>
+        A value of <code>0</code> will cause any <a>NEL policy</a> for this
+        <a>origin</a> to be removed from the <a>policy cache</a>.
+        </p>
+
+        <p class="note">
+        To ensure delivery of NEL reports, the <a>server</a> should ensure that
+        the Reporting <a>endpoint group</a> is also configured with a
+        sufficiently high <code>max_age</code>.  If the Reporting policy
+        expires, NEL reports will not be delivered, even if the NEL policy has
+        not expired.
+        </p>
+      </section>
+
+      <section>
+        <h2>The <code>include_subdomains</code> member</h2>
 
         <p>
-        The user agent MAY extend the above error type list with custom values —
-        e.g. new error types to accommodate new protocols, or more detailed
-        error descriptions of existing ones. When doing so, the user agent
-        SHOULD follow the dot-delimited pattern
-        (<code>[group].[optional-subgroup].[error-name]</code>) to facilitate
-        simple and consistent processing of the error reports — e.g. the
-        collector may provide aggregation by category and/or one or multiple
-        subgroups.
+        The OPTIONAL <dfn><code>include_subdomains</code></dfn> member is a
+        boolean that enables this <a>NEL policy</a> for all subdomains of this
+        origin.  If no member named <code>include_subdomains</code> is present
+        in the object, or its value is not <code>true</code>, the <a>NEL
+        policy</a> will not be enabled for subdomains.
         </p>
-        </dd>
-      </dl>
+
+        <p class="note">
+        To ensure delivery of NEL reports for subdomains, the application should
+        ensure that the Reporting <a>endpoint group</a> is also configured with
+        <code>include_subdomains</code> enabled.  If the Reporting policy is
+        not, and there is not a separate Reporting policy for a given subdomain,
+        NEL reports for that subdomain will not be delivered, even if the NEL
+        policy includes the subdomain.
+        </p>
+      </section>
+
+      <section>
+        <h2>The <code>success_fraction</code> member</h2>
+
+        <p>
+        The OPTIONAL <dfn><code>success_fraction</code></dfn> member defines the
+        <a>sampling rate</a> that should be applied to reports about
+        <a>successful</a> <a>network requests</a> for this origin.  If present,
+        its value MUST be a number between <code>0.0</code> and
+        <code>1.0</code>, inclusive; any other value will result in a parse

```

Comment:
Parse error? As in, the whole registration is aborted? Perhaps, "any other value will be ignored"?

Reply:
This is consistent with what we do in Reporting.  If you leave an optional field out, we provide a suitable default, but if you provide the field with an invalid value, we consider the entire policy invalid.  If we want to change this I think we should update Reporting at the same time, and it should be in a separate PR.

Reply:
Hmm, I'm generally in favor of "be liberal in what you accept" principle.. unless the fallback runs the risk of some security / other exposure. In this case, seems like the fallback would be "0.0", which is ~safe?